src/Documentation/SidebarMenu/SidebarMenuHelper.js

```diff
@@ -0,0 +1,161 @@
+import startCase from 'lodash.startcase'
+import kebabCase from 'lodash.kebabcase'
+import sidebar from '../sidebar'
+
+export const PATH_SEPARATOR = '/'
+export const PATH_TO_DOC = '/doc'
+
+export default class SidebarMenuHelper {
+  static fillFilesArray(section, file, arr) {
+    let folder = SidebarMenuHelper.getParentFolder(file, section)
+    let filename = SidebarMenuHelper.extractFilename(file)
+    let path = SidebarMenuHelper.getFullPath(folder, filename)
+    arr[path] = startCase(
+      SidebarMenuHelper.removeExtensionFromFileName(filename)
+    )
+  }
+
+  static isString = obj => typeof obj === 'string'
+
+  static transformAbsoluteToDocRelatedPath = path =>
+    path
+      .join(PATH_SEPARATOR)
+      .replace(new RegExp(PATH_TO_DOC, 'i'), '')
+      .split(PATH_SEPARATOR)
+      .filter(_ => _)
+
+  static combineToPath = subPaths => [].concat(subPaths).join(PATH_SEPARATOR)
+
+  static getZeroFile = arr => {
+    const firstItem = arr[0]
+    const { files, indexFile } = firstItem
+    return (
+      (files && SidebarMenuHelper.getZeroFile(files)) || indexFile || firstItem
+    )
+  }
+
+  static findFileByName = (item, find) => {
+    let file = null
+    if (
+      SidebarMenuHelper.removeExtensionFromFileName(item.indexFile || item) ===
```

Comment:
install pre-commit hook and/or run prettier, it's > 80 symbols

Reply:
his installed